### ISSUE TITLE
Mann Accessibility Fixes

### ISSUE THREAD

**Issue by sarah-cul** [cul-it/uls#1212]
siteimprove score is currently at 86.8. To improve this score:

found with ANDI:
- [x] image of staff profiles on home page, needs accessible name/title
- [x] image of laptops & equipment, needs accessible name/title
- [x] About page, there are 2 virtual exhibits links going to different places. The link on the About page goes to the exhibits spotlight page, and the menu item under News & Events -> Virtual Exhibits goes to https://mann.library.cornell.edu/virtual-exhibits. Confirmed with content owner, us the intermediary page on the Mann website not spotlight.
- [x]  About -> Friends of Mann. The links for Events Calendar need to be updated to point https://mann.library.cornell.edu/events-calendar.
- [x]  Full Hours links ambiguous Banner & menu 
- [x] Home menu & footer ambiguous links
- [x] News & Events menu link and footer link, ambiguous. footer linked page appears dated. 

- [x] Collections, 4 links ambiguous for all collections in the menu and on the page.
- [x] News & Events, 4 links ambiguous for all links in the menu and on the page.
- [ CAUSED BY LOCALIST WIDGET] News & Events, the selection under the Calendar [Select has no accessible name, associated <label>, or [title]
- [ CAUSED BY LOCALIST WIDGET ]  Events Calendar, the links to Location: Mann Library are all ambiguous
- [  CAUSED BY LOCALIST WIDGET]  News & Events -> Virtual Exhibits. The Filter by section has anchor tags without href, need role=link
